solr/solrj/src/java/org/apache/solr/client/solrj/impl/ConcurrentUpdateSolrClient.java

```diff
@@ -100,8 +101,13 @@ public class ConcurrentUpdateSolrClient extends SolrClient {
    */
   protected ConcurrentUpdateSolrClient(Builder builder) {
     this.internalHttpClient = (builder.httpClient == null);
+    final var hscBuilder =
```

Comment:
might just be me, but `hscBuilder` didn't really ring for me...   `builder`?   `httpSolrClientBuilder` ?    I don't love the initials style...

Reply:
Done!  I had to use `httpSolrClientBuilder`, because there's already a 'builder' variable in scope here.